x/authenticator/types/msgs.go

```diff
@@ -42,6 +42,7 @@ func (msg *MsgAddAuthenticator) GetSigners() []sdk.AccAddress {
 var _ sdk.Msg = &MsgRemoveAuthenticator{}
 
 func (msg *MsgRemoveAuthenticator) ValidateBasic() error {
+	// TODO: call validate here
```

Comment:
I don't think we can call validate here without a redundant call to `m.Keeper.GetAuthenticatorsForAccount(ctx, sender)`

Reply:
I think we could use `authenticator.Validate(msg.Sender, msg.Data)`, when it's written